redis_info.py

```diff
@@ -202,10 +246,52 @@ def dispatch_value(info, key, type, plugin_instance=None, type_instance=None):
     # write_http plugin. See
     # https://github.com/collectd/collectd/issues/716
     val.meta = {'0': True}
+    val.dispatch()
+
+
+def dispatch_llen_key(key_name, key_value, db_index, plugin_instance):
```

Comment:
Reuse the original dispatch_value function and just pass in the dimensions as another param.  The way it is here there is a lot of duplication of code.